core/src/main/java/com/xmartlabs/base/core/module/GsonModule.java

```diff
@@ -68,12 +67,13 @@ public boolean shouldSkipClass(Class<?> clazz) {
 
   @NonNull
   @Provides
-  GsonBuilder provideCommonGsonBuilder() {
-    GsonBuilder gsonBuilder = new GsonBuilder();
-    if (BuildConfig.DEBUG) {
-      gsonBuilder.setPrettyPrinting();
```

Comment:
The idea was for the user to modify it in `modifyGsonBuilder`, but I agree, let's add it as default there, as we will probably always use it.